### PR TITLE
Fixes #1224 : Hi-Fi drag drop interaction

### LINKED ISSUE #1224: HighFi : Drag Drop Interaction
Hifi Drag-drop interaction which includes improving the view aspects and polishing the drag-drop functionality

**Target PR date**: 3 June 2020
**Target completion date**: 6 June 2020

### PR DESCRIPTION
**Pull request by aggarwalpulkit596**
## Explanation

Fixes #1224 

In this PR I've just finalized the view by doing some hi-fi stuff to the views.

Screenshot_1593326140
Screenshot_1593326130
Screenshot_1593326149
Screenshot_1593326198
Screenshot_1593326218

Mocks -> https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/35409b49-f76f-49ad-8de9-f2e883d77880/PM-Q8-Grouping-Drag-Drop-No-Answer-11
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by vinitamurthi — APPROVED** [review]
Please wait for approval from either @rt4914 or @nikitamarysolomanpvt

**Review by anandwana001 — APPROVED** [review]
LGTM

File: app/src/main/res/drawable/rounded_rect_grey_border_gradient_white_background.xml
Comment (rt4914) [inline app/src/main/res/drawable/rounded_rect_grey_border_gradient_white_background.xml]: Nit suggested:
```
    android:centerColor="#F7F7F7"
    android:endColor="#DCDCDC"
    android:startColor="#FFFFFF"
```

Suggested `F7` because it is the color in between `FE` and `EF`

**Review by rt4914 — APPROVED** [review]
LGTM, I have not given very high priority to its highfi because its out margin/padding depends on other items too and therefore this will be done as a part of StateFragment highfi. Else internally everything is great.

File: app/src/main/res/drawable/rounded_rect_grey_border_gradient_white_background.xml
Comment (aggarwalpulkit596) [reply app/src/main/res/drawable/rounded_rect_grey_border_gradient_white_background.xml]: done

**Pull request merged**
